api/app/tests/apps/api/v1/request_data/create_initial.json

```diff
@@ -22,7 +22,7 @@
         "extra_properties": {}
     },
     "category": {
-        "sub_category": "http://example.com/placeholder/replace/in/tests"
+        "sub_category": "/signals/v1/public/terms/categories/overig/sub_categories/overig"
```

Comment:
Leave the non-existing category - we should not have an existing category here (because overig/overig may also be removed at some point in future).

Reply:
Did this to improve the performance. This way we do not need to create a new category, the assumption is that the "overig/overig" category will exists. If it is removed in the future the tests will fail and we can change the value

Reply:
Created a ticket specifically for this point (SIG-1531). The test suite should not contain hardcoded categories (unless very specific logic is to be tested).